pkg/controller/secret/secret_controller.go

```diff
@@ -260,6 +283,35 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 	}
 }
 
+// createOCMAgentRoute creates an AlertManager Route for OcmAgent in memory.
+func createOCMAgentRoute() *alertmanager.Route {
+	return &alertmanager.Route{
+		Receiver: receiverOCMAgent,
+		Continue: true,
```

Comment:
Should this be false?

Reply:
It ultimately doesn't matter as the same alerts get nullified in the pagerduty routing block, but I can't think of any danger of setting it to false either. I'll test this and update if AM's behaviour remains as expected.